src/components/mixins/control.ts

```diff
@@ -135,9 +135,11 @@ export default class ControlMixin extends Vue {
     }
 
     doSendMove(gcode: string, feedrate: number) {
-        gcode = 'G91' + '\n' + 'G1 ' + gcode + ' F' + feedrate * 60
-
-        if (this.absolute_coordinates) gcode += '\nG90'
+        gcode =
+            `SAVE_GCODE_STATE NAME=ui_movement\n` +
```

Comment:
I know there is no value in this change, but keeping consistency, I would argue that the name should be prefixed with "_" to mark it as "internal".

```suggestion
            `SAVE_GCODE_STATE NAME=_ui_movement\n` +
```

FWIW, Klipper does exactly this internally on [firmware_retraction](https://github.com/Klipper3d/klipper/blob/08a1c9f12760ee6d89db2b82e76c7d93453212db/klippy/extras/firmware_retraction.py#L56) module.